Makefile

```diff
@@ -205,11 +205,22 @@ docker: _print_vars
 	docker build -t $(IMG) -f ./build/Dockerfile .
 	docker push $(IMG)
 
+### gen_webhook: generate cert manager certificates for kubernetes webhooks and deploy to cluster; no-op if running on OpenShift
+gen_webhook:
+ifeq ($(WEBHOOK_ENABLED),true)
+ifeq ($(TOOL),kubectl)
+	$(eval IS_NOT_FOUND := $(shell kubectl get namespace cert-manager 2>&1 | grep -q "not found"; echo $$?))
+	@if [ "$(IS_NOT_FOUND)" = "0" ]; then echo "Cert Manager has not been found."; else kubectl apply -f deploy/k8s_webhook/cert_manager.yaml; fi
```

Comment:
This makes our makefile single-namespace again; I think we need something like 
```bash
sed -i.bak -e "s|namespace: .*|namespace: $(NAMESPACE)|" \
           -e "s|devworkspace-webhookserver.devworkspace-controller.svc|devworkspace-webhookserver.$(NAMESPACE).svc|' \
            deploy/k8s_webhook/cert_manager.yaml
kubectl apply -f deploy/k8s_webhook/cert_manager.yaml -n $(NAMESPACE)
mv deploy/k8s_webhook/cert_manager.yaml.bak deploy/k8s_webhook/cert_manager.yaml
```
which is a pain (as usual).

I think we should also split this onto multiple lines; I'm not fully sure the MacOS `make` version won't cause issues but if it does we should just do line continuations (`\`)